lib/commands/commit.js

```diff
@@ -48,12 +48,18 @@ module.exports = {
       return runCommand('git add . && git commit -m "' + options.message + '"', execOptions)
     }
 
+    function returnToPreviousCheckout() {
+      // Each '\' needed to be escaped
+      return runCommand("git checkout `git reflog HEAD | sed -n '/checkout/ !d; s/.* \\(\\S*\\)$/\\1/;p' | sed '2 !d'`", execOptions);
+    }
+
     return buildApp()
       .then(checkoutGhPages)
       .then(copy)
-      .then(addAndCommit).then(function() {
+      .then(addAndCommit)
+      .then(returnToPreviousCheckout).then(function() {
         var branch = options.branch;
-        ui.write('Done. All that\'s left is to git push the ' + branch + ' branch.\n');
+        ui.write('Done. All that\'s left is to git push the ' + branch + ' branch.\nEx: git push origin ' + branch + '\n');
```

Comment:
Please place the `.then(function() {` on the following line and wrap/concatenate the string before column 80.